src/chess/net/net_stockfish.rs

```diff
@@ -0,0 +1,1194 @@
+const PASSED_FILE: [Score; FILE_CNT] = [
+    Score(9, 10),
+    Score(2, 10),
+    Score(1, -8),
+    Score(-20, -12),
+    Score(-20, -12),
+    Score(1, -8),
+    Score(2, 10),
+    Score(9, 10),
+];
+
+//const PASSED_RANK: [Score; FILE_CNT] = [
+//    Score(0, 0), Score(5, 7), Score(5, 13), Score(32, 42),
+//    Score(70, 70), Score(172, 170), Score(217, 269), Score(0, 0)
+//];
+
+const PASSED_DANGER: [i32; RANK_CNT] = [0, 0, 0, 2, 7, 12, 19, 0];
+
+// Assorted bonuses and penalties used by evaluation
+const MINOR_BEHIND_PAWN: Score = Score(16, 0);
+const BISHOP_PAWNS: Score = Score(8, 12);
+const CONNECTIVITY: Score = Score(3, 1);
+const LONG_RANGED_BISHOP: Score = Score(22, 0);
+const ROOK_ON_PAWN: Score = Score(8, 24);
+const TRAPPED_ROOK: Score = Score(92, 0);
+const WEAK_QUEEN: Score = Score(50, 10);
+const CLOSE_ENEMIES: Score = Score(7, 0);
+const PAWNLESS_FLANK: Score = Score(20, 80);
+const THREAT_BY_SAFE_PAWN: Score = Score(192, 175);
+const THREAT_BY_RANK: Score = Score(16, 3);
+const HANGING: Score = Score(48, 27);
+const WEAK_UNOPOSSED_PAWN: Score = Score(5, 25);
+const SLIDER_ON_QUEEN: Score = Score(42, 21);
+const THREAT_BY_PAWN_PUSH: Score = Score(38, 22);
+// const THREAT_BY_ATTACK_ON_QUEEN: Score = Score(38, 22);
+const HINDER_PASSED_PAWN: Score = Score(7, 0);
+// const TRAPPED_BISHOP_A1H1: Score = Score(50, 50);
+
+// Penalties for enemy's safe checks
+const QUEEN_SAFE_CHECK: i32 = 780;
+const ROOK_SAFE_CHECK: i32 = 880;
+const BISHOP_SAFE_CHECK: i32 = 435;
+const KNIGHT_SAFE_CHECK: i32 = 790;
+
+const LAZY_THRESHOLD: Value = 1500;
+const SPACE_THRESHOLD: Value = 12222;
+
+// #[repr(u8)]
+// #[derive(Copy, Clone)]
+// enum EvalPasses {
+//     Pawn = 0,
+//     Knight = 2,
+//     Bishop = 3,
+//     Rook = 4,
+//     Queen = 5,
+//     King = 6,
+//     Material = 8,
+//     Imbalance = 9,
+//     Mobility = 10,
+//     Threat = 11,
+//     Passed = 12,
+//     Space = 13,
+//     Initiative = 14,
+//     Total = 15,
+// }
+
+// const EVAL_PASSES_CNT: usize = 16;
+
+// struct PassScore {
+//     pass: EvalPasses,
+//     score_white: Score,
+//     score_black: Score,
+// }
+
+// impl fmt::Display for PassScore {
+//     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+//         match self.pass {
+//             EvalPasses::Material
+//             | EvalPasses::Imbalance
+//             | EvalPasses::Initiative
+//             | EvalPasses::Total => write!(f, " ----  ---- | ----  ---- ")?,
+//             _ => write!(f, "{} | {}", self.score_white, self.score_black)?,
+//         }
+//         write!(f, " | {}", self.score_white - self.score_black)
+//     }
+// }
+
+pub struct Evaluation {}
+
+impl Evaluation {
+    pub fn evaluate(board: &Board) -> Value {
+        let mut pawn_entry = PawnEntry::new(board);
+        let mut material_entry = MaterialEntry::new(board);
+        let mut eval = EvaluationInner::new(board, &mut pawn_entry, &mut material_entry);
+        eval.value()
+    }
+

```

Comment:
What's this section about? Why is it commented out?

Reply:
moved to utils.rs before, removed